README.md

```diff
@@ -67,17 +67,33 @@ artifact.platform # => "ubuntu"
 artifact.platform_version # => "14.04"
 ```
 
+### List the available versions for a product and channel
+```ruby
+# Note that this feature currently only works for :unstable channel
+options = {
+  channel: :unstable,
+  product_name: 'chef',
+}
+
+Mixlib::Install.new(options).available_versions
+
+# => ["12.13.3", "12.13.7", "12.13.8+20160721014124", "12.13.11+20160721165202"]
+```
+
 ## Unstable channel
 The `:unstable` channel is currently only available when connected to Chef's internal network.
 
+## Feature Flags
+
+Below are the environment variables you can set to tune certain aspects of mixlib-install. They enable some features that are currently under development. You should use these features with care.
+
+`FULL_ARTIFACTORY` => Enables Artifactory backend for all available channels.
```

Comment:
Since env vars are the ultimate global namespace, we should use an env vars prefixed with `MIXLIB_INSTALL_*`. Maybe `MIXLIB_INSTALL_FEATURE_UNIFED_BACKEND` or similar.